kubernetes/cray-sts/Chart.yaml

```diff
@@ -1,5 +1,29 @@
-apiVersion: v1
-description: "Kubernetes resources for cray-sts"
-name: "cray-sts"
-home: "cloud/cray-sts"
-version: 0.6.0
+apiVersion: v2
+name: cray-sts
+version: 0.7.0
+description: Kubernetes resources for cray-sts
+keywords:
+  - cray-sts
+home: https://github.com/Cray-HPE/cray-sts
+dependencies:
+  - name: cray-service
+    version: ~7.0.0
+    repository: https://artifactory.algol60.net/artifactory/csm-helm-charts/
+maintainers:
+  - name: kimjensen-hpe
+  - name: bklei
+  - name: brantk-hpe
+appVersion: 0.7.0
```

Comment:
The appVersion needs to be set from the VERSION = getDockerBuildVersion(isStable: env.IS_STABLE) that was set in the Jenkinsfile.github. This is how the unstable builds reference the image that was built.

Reply:
This is handled by the `chart-metadata` target when running `make chart`. You can verify this locally by running e.g., `VERSION=9.9.9-beta.9 make image chart` and examining the resulting artifacts (though the working tree will also be dirty):

```
$ helm show chart packages/cray-sts-9.9.9-beta.9.tgz 
annotations:
  artifacthub.io/changes: |
    - kind: security
      description: Updated images to CSM patched versions
      links:
        - name: Github Issue
          url: https://github.com/Cray-HPE/cray-sts/issues/12
        - name: Github PR
          url: https://github.com/Cray-HPE/cray-sts/pull/11
  artifacthub.io/images: |
    - name: cray-sts
      image: artifactory.algol60.net/csm-docker/stable/cray-sts:9.9.9-beta.9
  artifacthub.io/license: MIT
apiVersion: v2
appVersion: 9.9.9-beta.9
dependencies:
- name: cray-service
  repository: https://artifactory.algol60.net/artifactory/csm-helm-charts/
  version: ~7.0.0
description: Kubernetes resources for cray-sts
home: https://github.com/Cray-HPE/cray-sts
keywords:
- cray-sts
maintainers:
- name: kimjensen-hpe
- name: bklei
- name: brantk-hpe
name: cray-sts
version: 9.9.9-beta.9
```

The expectation is that the chart metadata should always be current, therefore it is recommended to run `make chart-metadata` whenever you change `.version` to automatically update the chart metadata (e.g., `version`, `appVersion`) as well as `global` settings in values.yaml and then commit the results.